src/transformers/user.js

```diff
@@ -177,30 +178,42 @@ export default (request, context) => {
         id: id.toString('base64')
       };
 
-      if (body.data.getUserStatusResponse.userName) {
+      if (get(body, 'data.getUserStatusResponse.userName', false)) {
         data.name = body.data.getUserStatusResponse.userName.$;
       }
-      if (body.data.getUserStatusResponse.userAddress) {
+      if (get(body, 'data.getUserStatusResponse.userAddress', false)) {
         data.address = body.data.getUserStatusResponse.userAddress.$;
       }
-      if (body.data.getUserStatusResponse.userPostalCode) {
+      if (get(body, 'data.getUserStatusResponse.userPostalCode', false)) {
         data.postalCode = body.data.getUserStatusResponse.userPostalCode.$;
       }
-      if (body.data.getUserStatusResponse.userMail) {
+      if (get(body, 'data.getUserStatusResponse.userMail', false)) {
         data.mail = body.data.getUserStatusResponse.userMail.$;
       }
 
       if (selectUserInfo.includes('userLoan')) {
         let loans = [];
-        if (body.data.getUserStatusResponse.userStatus.loanedItems.loan) {
+        if (
+          get(
+            body,
+            'data.getUserStatusResponse.userStatus.loanedItems.loan',
+            false
+          )
+        ) {
           loans = body.data.getUserStatusResponse.userStatus.loanedItems.loan;
         }
```

Comment:
Det virker lidt som dobbeltkonfekt at bruge lodash/get til at tjekke om en værdi findes og at sætte den direkte
```suggestion
        const loans =  get(
            body,
            'data.getUserStatusResponse.userStatus.loanedItems.loan',
            []
          )
```